packages/adapters/bitcoin/src/utils/HelperUtil.ts

```diff
@@ -1,11 +1,17 @@
 import { BitcoinNetworkType } from 'sats-connect'
 
-export function mapCaipNetworkToXverseName(caipNetworkId: string): BitcoinNetworkType {
+import { bitcoin, bitcoinTestnet } from '@reown/appkit/networks'
+
+export function mapCaipNetworkToXverseName(caipNetworkId: string | undefined): BitcoinNetworkType {
```

Comment:
this sounds like very xverse specific, why not on its connector directly?

Reply:
Thought better to keep helpers/utils to separate as we follow this convention in different places 🤔